File: lab03/lab03_task01/src/lab03_task01_srv_node.cpp
#include "ros/ros.h"
//TODO: Include the header file rotate_point service (look in the package example_msg_srv)

//TODO: Complete this function
bool rotate_point_3d();

int main(int argc, char **argv)
{

    ros::init(argc, argv, "rotate_point_server");

    ros::NodeHandle nh;

    //TODO: Initialise the service 'rotate_point' and have it calls the function 'rotate_point_3d'
    ros::spin();

    return 1;
}

File: lab03/lab03task01/scripts/lab03_task01_srv_node.py
#!/usr/bin/env python

import rospy
import numpy as np
##TODO: Include the header file rotate_point service (look in the package example_msg_srv)

##TODO: Complete this function
def rotate_point_3d():
    print 'test'


def rotate_point_server():
    rospy.init_node('rotate_point_server')
    ##TODO: Intialise the service 'rotate_point' and have it calls the function 'rotate_point_3d'
    rospy.spin()


if __name__ == "__main__":
    rotate_point_server()